index.html

```diff
@@ -9,11 +9,12 @@
 <body>
   <h1>Awesome Books</h1>
   <div id="book-list"></div>
-  <form id="add-book-form">
-    <input type="text" id="title" name="title" placeholder="Title" required><br><br>
-    <input type="text" id="author" name="author" placeholder="Author" required><br><br>
-    <button type="submit">Add</button>
-  </form>
+
+    <form id="add-book-form">
+      <input type="text" id="title-input" placeholder="Title" required>
+      <input type="text" id="author-input" placeholder="Author" required>
+      <button type="submit">Add Book</button>
+    </form>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md), kindly use CSS styles to make your application match [this wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) 👍

Your design
![image](https://github.com/Jessekay/AwesomeBooks/assets/78251927/83611a05-6f92-459a-b291-8d2764517d45)